client/src/pages/Login.tsx

```diff
@@ -52,6 +56,16 @@ const Login: React.FC = () => {
         setMessage
     ] = useState<string>('');
 
+    const [
+        cookies,
+        setCookie,
+    ] = useCookies(['token']);
```

Comment:
Probably should put this in a utils file so that it doesn't have to be declared multiple times

Reply:
I did not know how to do that, please kindly pull my code on your local branch, try to fix it and i'll gladly accept the changes

Reply:
Nvm, my idea didn't work out